lib/rucio/daemons/conveyor/poller.py

```diff
@@ -72,7 +72,14 @@ def poller(once=False, activities=None, sleep_time=60,
     except NoOptionError:
         timeout = None
 
-    executable = ' '.join(sys.argv)
+    executable = sys.argv[0]
+    if activities:
+        activities.sort()
```

Comment:
does this really come as a list? i think (may be mistaken) that this is just a single string, so you're sorting the characters in the string here?